### ISSUE TITLE
Multiple snap points of one svg rect?

### ISSUE THREAD

**Issue by midl0** [taye/interact.js#132]
Hello,

first of all i want to say i really like your work. I am new to javascript so i might be missing something.
I already made my first goal working - svg inside html to position and rotate rectangles of variable sizes which represent individual cabinets in room. Grid snaping works just fine.

So to get to my problem. I have svg element inside which are rectangles of variable sizes, positions and rotations. Is it possible to have multiple elementOrigin for snapping, eg. to not to only snap by origin point of rectangle but by other three corners as well?

Basically is possible to make two rectangles with some rotations "snapable" BY all 4 corners of moving rect TO all 4 corners of target rect??

I found examples for my second related question about interaction with other rectangles snaping to multiple anchors and all their corners, so i think i could manage this one on my own. I am just not sure how to implement "the moving rectangle" snapping.

I would appreciate If you could just point me somewhere to look for the answer or tell me if it is even possible. Thanks
